math/pythagoras/pythagoras.go

```diff
@@ -0,0 +1,19 @@
+package pythagoras
+
+import (
+	"math"
+)
+
+//Vector defines a tuple with 3 values in 3d-space
+type Vector struct {
+	x float64
+	y float64
+	z float64
+}
+
+//Distance calculates the distance between to vectors with the   Pythagoras theorem
+func Distance(a, b Vector) float64 {
+	abs := float64(math.Abs(b.x - a.x))
```

Comment:
Why are calculating absolute value for the x component. We are guaranteed to get a positive number when we square it.